plugins.json

```diff
@@ -23073,6 +23073,80 @@
+      "release_notes_url": "https://github.com/mattermost/mattermost-plugin-channel-export/releases/tag/v1.2.1",
+      "version": "1.2.1",
+      "min_server_version": "5.37.0",
+      "server": {
+        "executables": {
+          "darwin-amd64": "server/dist/plugin-darwin-amd64",
+          "darwin-arm64": "server/dist/plugin-darwin-arm64",
+          "linux-amd64": "server/dist/plugin-linux-amd64",
+          "linux-arm64": "server/dist/plugin-linux-arm64",
+          "windows-amd64": "server/dist/plugin-windows-amd64.exe"
+        },
+        "executable": ""
+      },
+      "webapp": {
+        "bundle_path": "webapp/dist/main.js"
+      },
+      "settings_schema": {
+        "header": "",
+        "footer": "",
+        "settings": [
+          {
+            "key": "EnableAdminRestrictions",
+            "display_name": "Enable Admin Restrictions",
+            "type": "bool",
+            "help_text": "Restricts the exporting of channels to system administrators or channel administrators",
+            "placeholder": "",
+            "default": false,
+            "hosting": "",
+            "secret": false
+          },
+          {
+            "key": "MaxFileSize",
+            "display_name": "Maximum size of channel export file in bytes",
+            "type": "number",
+            "help_text": "Determines the maximum size of the channel export file when using the slash command. A value of 0 will use the [FileSettings.MaxFileSize](https://docs.mattermost.com/configure/environment-configuration-settings.html#maximum-file-size) from Mattermost server.",
+            "placeholder": "",
+            "default": 0,
+            "hosting": "",
+            "secret": false
+          }
+        ],
+        "sections": null
+      }
+    },
+    "platforms": {
+      "linux-amd64": {
+        "download_url": "https://plugins.releases.mattermost.com/release/mattermost-plugin-channel-export-v1.2.1-linux-amd64.tar.gz",
+        "signature": "iQIzBAABCAAdFiEExViBuA9p6GO4WtXR0bVLR6XO/sQFAmdiPMgACgkQ0bVLR6XO/sQB7A/+MC1vTC0LD8Mr0SbPXHDqPH08jqs5pUxcF3DDG9zWq3Haesm3+bMA50bJKFIC/lfVzA3UPYEwRzxgo+g4Zb4wmm+ZdO27yH30jTWqOlx33nxQeq+UcgKFuDvH/B5rUHXpkG0evmSOSEgnsuvizIwJaup6BqktlQyx2hS5Q7nzMw2ZKBeGwv68T//l/evIsn7St0Y4ZbgTq1uHzRmfyWd4tzBlPsThBkPQx+HmI7y4dPBj4Dp7XAhiUv2F9nQnZrNAyNuDrYw7nVigF/BRJmpZRweChOMBJ+VI6E/D4Jx8IADu/72PoIlV4dGR3JeHYq7zKsT3RqrDsE2G4bjaZxQNrvvYJ8W3QpEHlLdspMe7l1bwcbDrz+JyC8Zi3ZsQIn+OamRQGVgDL4VLmu8YGgJbVYzKLDaAASh7StRsaP75iplF/IKMD2KLAmjCx1sCzZfwo30hIsDzwyY/PQk4ffA2coXCaC1XRTecG+HQxzhicg3v8ne4pcI13QJDl5p6Ov5zute7NIlE/nKy3NZou/x3BJm8xxsyy3CNvdCSdhBAWBc+l2eGgyXnw8nMSEgCgObmqFzN+OSqtwqFFV0UCN8T87Vrx3kzU/YW6EnGcJJ3Mb877V5RbamMpLe74fCK0xbRhQFovsUqv55pZR3obIm02FxZekM309F5d98NKQscYKg="
+      },
+      "darwin-amd64": {},
+      "windows-amd64": {
+        "download_url": "https://plugins.releases.mattermost.com/release/mattermost-plugin-channel-export-v1.2.1-windows-amd64.tar.gz",
```

Comment:
This usually is a zip file

Reply:
Is it? At least the GitHub plugin is also using `tar.gz`: https://github.com/mattermost/mattermost-marketplace/pull/472/files#diff-18d7f9b905f83805c42db6d239a7fb0207d4b32193202eb13ba077caed424c3cR23353

Reply:
I think a bigger issue is, why are we publishing Windows and Mac versions of plugins when we don't support servers on those platforms?  That's a discussion for another time.  @toninis this plugin has been published as tar.gz going back to 2020 (first release).